### PR TITLE
Clarify execute_result in messaging doc for IOPub

### PR DESCRIPTION
**Pull request by willingc**
Addresses #110.

### CODE REVIEW

File: docs/messaging.rst [hunk 1]
```diff
@@ -184,7 +184,7 @@ which is the topic for IOPub subscribers, e.g. ``execute_result``, ``display_dat
     In most cases, the IOPub topics are irrelevant and completely ignored,
     because frontends just subscribe to all topics.
     The convention used in the IPython kernel is to use the msg_type as the topic,
-    and possibly extra information about the message, e.g. ``execute_result`` or ``stream.stdout``
+    and possibly extra information about the message, e.g. ``kernel.{u-u-i-d}.execute_result`` or ``stream.stdout``
```
Comment (rgbkrk) [inline docs/messaging.rst]: Where's this uuid come from?

**Comment by rgbkrk** [comment]
I'll just merge this and we can feel free to iterate after.

File: docs/messaging.rst (hunk 1 above)
Comment (willingc) [inline docs/messaging.rst]: @rgbkrk See Issue #110. I believe the ipykernel is setting the uuid.

Comment (rgbkrk) [inline docs/messaging.rst]: Ah, k. Looks like I'll need to dig a little bit deeper later. I've been ignoring identities in my current code at the moment.

**Comment by takluyver** [comment]
Thanks @willingc!

**Comment by willingc** [comment]
Thank you @takluyver :cookie:

File: docs/messaging.rst (hunk 1 above)
Comment (takluyver) [inline docs/messaging.rst]: I think this affects all messages, not just `execute_result`. I.e. it's no longer 'possibly extra information' - it's always getting set to `kernel.{u-u-i-d}.msg_type` (and in ipyparallel, `engine.{u-u-i-d}.msg_type`).

**Comment by willingc** [comment]
And @rgbkrk :cookie:

**Pull request merged**